docs/cvl/methods.md

```diff
@@ -178,6 +190,63 @@ shown in the web report can indicate whether a summary was applied.
 
 ```
 
+(catch-unresolved-calls-entry)=
+### Catch unresolved-calls entry
+The catch unresolved-calls entry is a special type of summary declaration that instructs the Prover to replace calls to unresolved external function calls with a specific kind of summary, dispatch list.
```

Comment:
Good place to put a syntax example here to set the scene

Reply:
Ok